metricbeat/helper/openmetrics/openmetrics_test.go

```diff
@@ -97,8 +97,8 @@ metrics_one_count_total{name="john",surname="williams"} 2
 metrics_one_count_total{name="jahn",surname="baldwin",age="30"} 3
 `
 
-	openMetricsCounterKeyLabelWithNaNInf = `# TYPE metrics_one_count_errors counter
-metrics_one_count_errors{name="jane",surname="foster"} 1
+	openMetricsCounterKeyLabelWithNaNInf = `# TYPE metrics_one_count_errors_total counter
```

Comment:
According to [this](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#counter-1), counter types MUST have the suffix `_total`. I had to fix this to pass the test.